### DISCUSSION TITLE
Strapi

### DISCUSSION THREAD

**Discussion by wottpal** [coollabsio/coolify#3648] [Service Template Requests]
https://docs.strapi.io/dev-docs/deployment

**Comment by aybasaran** [comment]
@wottpal DONE! 
![{4A2F8FFD-E8AA-4235-8F65-3DA3428FA846}](https://github.com/user-attachments/assets/eeda1275-873d-46e5-bce0-53d2b91ec6dc)

**Comment by peaklabs-dev** [comment]
Strapi is now available as a one-click template.

**Reply by wottpal** [reply]
wow 🔥 ty so much

**Comment by xerxiastudios** [comment]
Thanks.  But I am getting This site can’t provide a secure connection. 
<img width="1051" alt="Screenshot 2024-11-05 at 12 57 08 PM" src="https://github.com/user-attachments/assets/75a5bc88-8446-41b6-8313-bab5b51ad4c2">
What can be the issue? Can you help please. @peaklabs-dev . Same with Directus I tried. Getting the same error. I am noob.

**Comment by flipch** [comment]
I don't think this is working properly as a one-click solution.

I believe both the development image is wrong to be used and traefik is only using http proxy and not https which blocks strapi's content policy.
